app/src/main/java/de/qspool/clementineremote/backend/player/MyLibrary.java

```diff
@@ -73,6 +75,8 @@
 
     private String mSelAlbum = "";
 
+    private boolean orderAlbumsByYear = false;
```

Comment:
Also, member variables in Android should be prefixed with `m`, like in the line above (`mSelAlbum`) :)